modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/internal/upgrade/v2_11_1/test/AccountRoleResourceUpgradeProcessTest.java

```diff
@@ -100,4 +132,10 @@ private void _runUpgrade() throws Exception {
 	)
 	private static UpgradeStepRegistrator _upgradeStepRegistrator;
 
+	@Inject
+	private EntityCache _entityCache;
+
+	@Inject
+	private FinderCache _finderCache;
+
```

Comment:
Position of these is due to SF specifically moving them, I'm assuming since they're private instead of private static.

Reply:
yes, it's because the other two are static